crates/egui/src/memory/mod.rs

```diff
@@ -1017,13 +1017,29 @@ impl OpenPopup {
 /// Only one can be open at a time.
 impl Memory {
     /// Is the given popup open?
-    pub fn is_popup_open(&self, popup_id: Id) -> bool {
+    ///
+    /// NOTE: To actually show the popup, you must call [`Self::show_popup`].
+    /// If [`Self::show_popup`] is not called for a frame, the open state will be cleared.
+    pub fn is_showing_popup(&self, popup_id: Id) -> bool {
         self.popups
             .get(&self.viewport_id)
             .is_some_and(|state| state.id == popup_id)
             || self.everything_is_visible()
     }
 
+    /// Check if the popup is open and keep it open.
+    pub fn show_popup(&mut self, popup_id: Id) -> bool {
```

Comment:
The docstring for `open_popup`  should link to this function, and vice-versa, to explain their difference..

Also: wouldn't it make sense if this called `self.open_popup`?